swagger-ui/how_to_run_dredd_tests.md

```diff
@@ -0,0 +1,20 @@
+Instructions on how to run the `dredd` tests
+
+1) Create the genesis state for the blockchain, start the blockchain and the rest server by running the following from the `swagger-ui` directory (`ctrc-c` to stop both the blockchain and the rest server):
```

Comment:
Mention that dredd needs to be installed globally using np,

Reply:
🆗